src/common/engine.ts

```diff
@@ -1022,6 +1022,30 @@ export class CurrencyEngine<
     return { edgeSpendInfo, nativeBalance, currencyCode, denom, skipChecks }
   }
 
+  async checkRecipientMinimumBalance(
+    balancePromise: (address: string) => Promise<string>,
+    sendAmount: string,
+    recipient: string
+  ): Promise<void> {
+    if (gte(sendAmount, this.minimumAddressBalance)) return
+
+    const balance = await balancePromise(recipient)
+    if (lt(add(sendAmount, balance), this.minimumAddressBalance)) {
+      const denom = this.currencyInfo.denominations.find(
+        denom => denom.name === this.currencyInfo.currencyCode
+      )
+      if (denom == null) throw new Error('Unknown denom')
+
+      const exchangeDenomString = div(
+        this.minimumAddressBalance,
+        denom.multiplier
+      )
+      throw new Error(
+        `Recipient address not activated. A minimum ${exchangeDenomString} ${this.currencyInfo.currencyCode} transfer is required to send funds to this address`
+      )
```

Comment:
We have a pending task in the core / GUI to add more `makeSpend` error types. I will mention this case in that task.